services/common/src/redux/slices/complianceReportsSlice.ts

```diff
@@ -3,129 +3,197 @@ import { ENVIRONMENT } from "@mds/common/constants/environment";
 import { createAppSlice, rejectHandler } from "@mds/common/redux/createAppSlice";
 import CustomAxios from "@mds/common/redux/customAxios";
 import * as API from "@mds/common/constants/API";
-import { IMineReportDefinition, IPageData, ItemMap } from "@mds/common/interfaces";
-import { createDropDownList, createItemMap, formatComplianceCodeReportName } from "@mds/common/redux/utils/helpers";
+import {
+  IMineReportDefinition,
+  IMineReportDueDateType,
+  IPageData,
+  ItemMap,
+} from "@mds/common/interfaces";
+import {
+  createDropDownList,
+  createItemMap,
+  formatComplianceCodeReportName,
+} from "@mds/common/redux/utils/helpers";
 import { createSelectorWrapper } from "../selectors/staticContentSelectors";
 import { createSelector } from "@reduxjs/toolkit";
 import { ISearchParams } from "@mds/common/interfaces/common/searchParams.interface";
-import { isEqual } from "lodash";
+import { notification } from "antd";
 
-
-export const complianceReportReducerType = 'complianceReports'
+export const complianceReportReducerType = "complianceReports";
 
 export interface ComplianceReportParams extends ISearchParams {
-    is_prr_only?: boolean[];
-    regulatory_authority?: "CPO" | "CIM" | "Both" | "NONE"[];
-    active_ind?: boolean[];
-    section?: string;
-};
+  is_prr_only?: boolean[];
+  regulatory_authority?: "CPO" | "CIM" | "Both" | "NONE"[];
+  active_ind?: boolean[];
+  section?: string;
+}
 
 export const reportParamsGetAll: ComplianceReportParams = {
-    active_ind: [true, false]
+  per_page: 0,
```

Comment:
I thought that I had made this `per_page` parameter unnecessary.

Reply:
Weird.  For the most part with the rebase I just accepted the incoming changes and reapplied my work where necessary.  Something must have gone wrong with it though as I didn't really touch any of this stuff.  I'll go over that again.